chain-tx-enclave-next/mls/src/keypackage.rs

```diff
@@ -81,6 +81,19 @@ impl KeyPackagePayload {
         ))
     }
 
+    /// insert or update extension
+    pub fn put_extension<T: MLSExtension>(&mut self, ext: &T) {
```

Comment:
pub fn put_extension(&mut self, ext: &MLSExtension) { 
is simpler

Reply:
Maybe use `pub fn put_extension(&mut self: ext: &impl MLSExtension)`?

Reply:
done